.github/workflows/scripts/release-single-plugin.sh

```
@@ -0,0 +1,104 @@
+echo "🔌 Releasing plugin: $PLUGIN_NAME"
+echo "🔧 Core version: $CORE_VERSION"
+
+PLUGIN_DIR="plugins/$PLUGIN_NAME"
+VERSION_FILE="$PLUGIN_DIR/version"
+
+if [ ! -f "$VERSION_FILE" ]; then
+  echo "❌ Version file not found: $VERSION_FILE"
+  exit 1
+fi
+
+PLUGIN_VERSION=$(cat "$VERSION_FILE" | tr -d '\n\r')
```

Shellcheck (0.10.0)

[style] Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead.

(SC2002)